backend/sumo-championship/src/main/java/com/sumoc/sumochampionship/db/season/Category.java

```diff
@@ -42,10 +45,16 @@ public class Category {
     private Season season;
 
     // Categories for a given tournament
-    @ManyToMany(mappedBy = "categories")
-    private Set<Tournament> tournaments;
+    // Changed from ManyToMany to ManyToOne
```

Comment:
Probably wrong comment, could be misleading later: later in code there is still ManyToMany

Reply:
Thanks. I will change it